pkg/auth/metrics_test.go

```diff
@@ -13,8 +13,12 @@ import (
 	"k8s.io/client-go/rest"
 )
 
+var (
+	internalproxyClientConfig = &rest.Config{}
```

Comment:
uppercase proxy? In this case you can also define that it doesn't configure anything like:

```suggestion
	defaultRestClientConfig = &rest.Config{}
```